### DISCUSSION TITLE
Error Message: Input, Output and Output_Maps folders in invalid working directory.

### DISCUSSION THREAD

**Discussion by Tom6161** [RangeShifter/RangeshiftR-tutorials#30] [Q&A] [answered]
Hi everyone,

When I try to run the simulation (in the tutorial, the first working example), I get this error message:

```
***** Invalid working directory: [my working directory path in my C drive]

***** Working directory must contain Inputs, Outputs and Output_Maps folders

*****
***** Simulation ABORTED 
*****
$Errors
[1] 666

```

I have folders in my working directory titled 'Inputs', 'Outputs' and 'Output_Maps'. Any ideas how to fix please?

I'm using R Studio:

RStudio 2022.02.3+492 "Prairie Trillium" Release (1db809b8323ba0a87c148d16eb84efe39a8e7785, 2022-05-20) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

With R Version 4.0.2

**Comment by JetteReeg** [comment, marked as answer]
Hi Tom,

the folders Inputs, Outputs and Output_Maps should be subfolders within a (sub)folder (_dirpath_) of your R working directory. 
In the case of the first working example, the subfolder inside the R working directory is called Overview (`dirpath = "Overview/"`) and inside this subfolder, the folders Inputs, Outputs and Output_Maps are created. When running RangeShiftR with the RunRS() command, the simulation parameters s and the RangeShiftR working directory _dirpath_ are handed over to RangeShiftR to start the simulation.

Could it be, that you have the three folders inside the R working directory and not inside another subfolder of the R working directory?

Best,
Jette

**Reply by Tom6161** [reply]
Thank you Jette, that works perfectly!

All the best
Tom
